integrations/spark/src/main/scala/marquez/spark/agent/plan/DatasetCollector.scala

```diff
@@ -0,0 +1,145 @@
+package marquez.spark.agent.plan
+
+import marquez.spark.agent.client.DatasetParser.DatasetParseResult
+import marquez.spark.agent.client.LineageEvent.{Dataset, DatasetFacet, SchemaDatasetFacet, SchemaField}
+import marquez.spark.agent.client.{DatasetParser, LineageEvent}
+import marquez.spark.agent.facets.OutputStatisticsFacet
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.catalyst.plans.logical._
+import org.apache.spark.sql.execution.QueryExecution
+import org.apache.spark.sql.execution.datasources.{HadoopFsRelation, InsertIntoDataSourceCommand, InsertIntoHadoopFsRelationCommand, LogicalRelation}
+import org.apache.spark.sql.execution.metric.SQLMetrics
+import org.apache.spark.sql.marquez.JDBCOperation
+import org.apache.spark.sql.types.StructType
+
+import java.net.URI
+import scala.collection.JavaConversions._
+
+class DatasetCollector {
```

Comment:
Is the Scala code used?

Reply:
Oops! That's what I get for not checking my own PRs 🤦🏽‍♂️